spec/lib/miq_automation_engine/models/miq_ae_class_compare_fields_spec.rb

```diff
@@ -1,4 +1,3 @@
-include MiqAeYamlImportExportMixin
```

Comment:
For this and every file with just this deletion, were these just copy/pasted in error and unnecessary in this file?